src/Init/Init.c

```diff
@@ -297,4 +263,3 @@ Init_setup_sdram_config(Sdramc* const sdramc)
 
     Sdramc_setConfig(sdramc, &IS42S16100H_SDRAM_CONFIG);
 }
-
```

Comment:
Empty line

Reply:
You mean an empty line is missing, right?

Reply:
It's all good everywhere now I think